ethicml/data/util.py

```diff
@@ -13,6 +16,27 @@
 ]
 
 
+def deprecated(func: Callable) -> Callable:
```

Comment:
Please use this, otherwise all type information about the decorated function is lost:

```suggestion
_F = TypeVar("_F", bound=Callable[..., Any])


def deprecated(func: _F) -> _F:
```